ui/apps/platform/src/Containers/MainPage/Banners/Banners.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react';
+
+import useCentralCapabilities from 'hooks/useCentralCapabilities';
+import useFeatureFlags from 'hooks/useFeatureFlags';
+import usePermissions from 'hooks/usePermissions';
+
+import AnnouncementBanner from './AnnouncementBanner';
+import CredentialExpiryBanner from './CredentialExpiryBanner';
+import DatabaseStatusBanner from './DatabaseStatusBanner';
+import OutdatedVersionBanner from './OutdatedVersionBanner';
+import ServerStatusBanner from './ServerStatusBanner';
+
+function Banners(): ReactElement {
+    // Assume MainPage renders this element only after feature flags and permissions are available.
```

Comment:
If this assumption becomes invalid, is the worst case that react will render `<></>` or equivalent?

Reply:
Yes, according to initial state of reducers. If I remember correctly, reasons to wait until after responses:
* Prevent flash of banners appearing after page renders.
* Probably also more determinate for integration tests.